testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -201,6 +204,196 @@ class TestPRMParser(TOPBase):
+        "bonds",
+        "angles",
+        "dihedrals",
+        "impropers",
+        "elements",
+        "chainIDs",
+    ]
+    expected_n_atoms = 677
+    expected_n_residues = 38
+    expected_n_segments = 3
+    expected_n_bonds = 695
+    expected_n_angles = 1220
+    expected_n_dihedrals = 1797
+    expected_n_impropers = 189
+    atom_i = 79
+    expected_n_zero_bonds = 4
+    expected_n_i_bonds = 3
+    expected_n_zero_angles = 9
+    expected_n_i_angles = 9
+    expected_n_zero_dihedrals = 14
+    expected_n_i_dihedrals = 15
+    expected_n_zero_impropers = 0
+    expected_n_i_impropers = 4
+    atom_zero_bond_values = ((0, 4), (0, 1), (0, 2), (0, 3))
+    atom_i_bond_values = ((79, 80), (79, 83), (77, 79))
+    atom_zero_angle_values = (
+        (0, 4, 6),
+        (0, 4, 10),
+        (3, 0, 4),
+        (2, 0, 3),
+        (2, 0, 4),
+        (1, 0, 2),
+        (1, 0, 3),
+        (1, 0, 4),
+        (0, 4, 5),
+    )
+    atom_i_angle_values = (
+        (80, 79, 83),
+        (77, 79, 80),
+        (77, 79, 83),
+        (74, 77, 79),
+        (79, 80, 81),
+        (79, 80, 82),
+        (79, 83, 84),
+        (79, 83, 85),
+        (78, 77, 79),
+    )
+    atom_zero_dihedral_values = (
+        (0, 4, 10, 11),
+        (0, 4, 10, 12),
+        (3, 0, 4, 5),
+        (3, 0, 4, 6),
+        (3, 0, 4, 10),
+        (2, 0, 4, 5),
+        (2, 0, 4, 6),
+        (2, 0, 4, 10),
+        (1, 0, 4, 5),
+        (1, 0, 4, 6),
+        (1, 0, 4, 10),
+        (0, 4, 6, 7),
+        (0, 4, 6, 8),
+        (0, 4, 6, 9),
+    )
+    atom_i_dihedral_values = (
+        (71, 74, 77, 79),
+        (74, 77, 79, 80),
+        (74, 77, 79, 83),
+        (75, 74, 77, 79),
+        (76, 74, 77, 79),
+        (77, 79, 80, 81),
+        (77, 79, 80, 82),
+        (77, 79, 83, 84),
+        (77, 79, 83, 85),
+        (78, 77, 79, 80),
+        (78, 77, 79, 83),
+        (80, 79, 83, 84),
+        (80, 79, 83, 85),
+        (81, 80, 79, 83),
+        (82, 80, 79, 83),
+    )
+    atom_zero_improper_values = ()
+    atom_i_improper_values = (
+        (74, 79, 77, 78),
+        (77, 80, 79, 83),
+        (79, 81, 80, 82),
+        (79, 84, 83, 85),
+    )
+    elems_ranges = [[0, 9], [250, 257], [500, 508]]
+
+    expected_elems = [
+        np.array(
+            [
+                "N",
+                "H",
+                "H",
+                "H",
+                "C",
+                "H",
+                "C",
+                "H",
+                "H",
+            ],
+            dtype=object,
+        ),
+        np.array(
+            [
+                "O",
+                "O",
+                "N",
+                "H",
+                "H",
+                "H",
+                "C",
+            ],
+            dtype=object,
+        ),
+        np.array(["H", "C", "O", "O", "N", "H", "H", "H"], dtype=object),
+    ]
+
+    expected_chainIDs = np.array(
+        [
+            "A",
+            "A",
+            "A",
```

Comment:
Is this "one line for each element" `black`'s doing? ... Not my favorite code aesthetic (because it makes it harder to see context) but I accept it as your preferred choice.

So don't worry about my comment, I am just venting — I actually recognize the upsides to auto-formatters, too.

Reply:
Yeah this is one of those bits that really kills readability and intent in code.

Reply:
I don't even understand the rationale when it seems perfectly capable to organize other arrays neatly fitted in a line. ... Aaaaanyways... I am trying out black in MDPOW, just to see what it feels like and if it's really as life-altering, time-saving as @RMeli says it is ;-).